categories.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+    <title>Quiz Mine</title>
+</head>
+<body>
+    <header class="header">
+        <div class="headertitle">Quiz Mine</div>
+        <div class="header-sub-title">Categories</div>
+    </header>
+    <main class="quiz-card-main">
+       <div class="quiz-card">
+           <div class="quiz-card-wrapper">
+               <img class="quiz-card-img" src="./assets/politics.jpg" alt="image not available" />
```

Comment:
You can give meaningful image descriptions in alt

Reply:
Updated meaningful descriptions in alt